Makefile

```diff
@@ -87,17 +87,17 @@ test: setup
 	$(AUTOPEP8) --in-place --aggressive --aggressive --recursive tests
 
 	# ensure pep8
-	# $(PEP8) storj
-	# $(PEP8) examples
-	# $(PEP8) tests
+	$(PEP8) storj
```

Comment:
no need for `PEP8` commands.
the command being executed by `tox` includes the execution of unit and integration tests + PEP8 checks + test coverage report.